apps/jenkins/jenkins/ptlsbox/jenkins-azure-vm-agent.yaml

```diff
@@ -29,73 +91,25 @@ spec:
                     deploymentTimeout: 1200
                     newResourceGroupName: "vm-agents-rg"
                     resourceGroupReferenceType: "new"
-                    maxVirtualMachinesLimit: 10
+                    maxVirtualMachinesLimit: 20
```

Comment:
you can just set this to 0 for no limit and then let the individual templates control it